nf_core/pipeline-template/lib/NfcoreTemplate.groovy

```diff
@@ -18,6 +18,20 @@ class NfcoreTemplate {
         }
     }
 
+    //
+    //  Warn if a -profile or Nextflow config has not been provided to run the pipeline
+    //
+    public static void checkConfigProvided(workflow, log) {
+        if (workflow.profile == 'standard' && workflow.configFiles.size() <= 1) {
+            log.warn "[$workflow.manifest.name] You are attempting to run the pipeline without any custom configuration!\n\n" +
+                    "This will be dependent on your local compute enviroment but can be acheived by:\n" +
+                    "   (1) Using an existing pipeline profile e.g. `-profile docker` or `-profile singularity`\n" +
+                    "   (2) Using an existing nf-core/configs for your Institution e.g. `-profile crick` or `-profile awsbatch`\n" +
```

Comment:
awsbatch is maybe a bit confusing here as not really an institution?

Reply:
Changed in https://github.com/nf-core/tools/pull/1309/commits/89af1d984b3eeefe3d8a985bcfe3abf4ee0d05e8